openforcefield/topology/topology.py

```diff
@@ -551,7 +551,7 @@ def from_dict(cls, d):
 # =============================================================================================
 
 
-class TopologyMolecule:
+class TopologyMolecule(object):
```

Comment:
I think inheriting from `object` is relevant only for Python 2. All classes in Python 3 should automatically inherit from `object` so that makes me think there may be an option somewhere in LGTM to set that we support only Python 3.

Reply:
I think this specifies how to make sure LGTM handles only Python 3: https://help.semmle.com/lgtm-enterprise/user/help/analysis-faqs.html#how-is-the-python-version-identified

Reply:
Thanks! I'll let you folks find that option.